var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -432,7 +454,6 @@ def configure_options(self):
                 ('scotch', 'ptscotch', True, True),
                 ('suite-sparse:umfpack,klu,cholmod,btf,ccolamd,colamd,camd,amd, \
                 suitesparseconfig,spqr', 'suitesparse', True, True),
-                ('hdf5:hl,fortran', 'hdf5', True, True),
```

Comment:
Perhaps this line can be retained - but the first string can be a `variable` that gets set appropriately [before this section]

something like
```
if '+fortran' in spec:
  hdf5libs = ':hl,fortran'
else:
  hdf5libs = ':hl'


                 ('hdf5'+hdf5libs, 'hdf5', True, True),

Reply:
Done.